core/src/main/scala/hammock/marshalling.scala

```diff
@@ -30,10 +29,10 @@ object marshalling {
 
   implicit def marshallC[F[_]](implicit I: InjectK[MarshallF, F]): MarshallC[F] = new MarshallC[F]
 
-  implicit def marshallNT[F[_]: Sync]: MarshallF ~> F = λ[MarshallF ~> F] {
+  implicit def marshallNT[F[_]: MonadThrow]: MarshallF ~> F = λ[MarshallF ~> F] {
     case um @ MarshallF.Unmarshall(entity) =>
       um.dec
         .decode(entity)
-        .fold(Sync[F].raiseError, Sync[F].pure)
+        .fold(MonadThrow[F].raiseError, MonadThrow[F].pure)
```

Comment:
Could be `liftTo` if syntax is imported
```suggestion
        .liftTo[F]
```